lib/teiserver_web/controllers/battle/match_controller.ex

```diff
@@ -132,4 +133,52 @@ defmodule TeiserverWeb.Battle.MatchController do
       |> render("ratings.html")
 
   end
+
+  @spec ratings_graph(Plug.Conn.t(), map) :: Plug.Conn.t()
+  def ratings_graph(conn, params) do
+
+    user = conn.assigns.current_user
+
+    filter = params["filter"] || "Team"
+    filter_type_id = MatchRatingLib.rating_type_name_lookup()[filter] || 1
+
+    ratings = Account.list_ratings(
+      search: [
+        user_id: user.id
+      ],
+      preload: [:rating_type]
+    )
+      |> Map.new(fn rating ->
+        {rating.rating_type.name, rating}
+      end)
+
+    logs = Game.list_rating_logs(
+      search: [
+        user_id: user.id,
+        rating_type_id: filter_type_id
+      ],
+      order_by: "Newest first",
+      limit: 50,
+      preload: [:match, :match_membership]
+    )
+
+    {dates, ratings_data} =
+      logs
+      |> List.foldl( %{}, fn rating, acc ->
```

Comment:
Putting spaces after the parenthesis isn't used here so the code looks all wrong.